control-plane-operator/controllers/hostedcontrolplane/v2/assets/kube-apiserver/deployment.yaml

```diff
@@ -276,6 +276,39 @@ spec:
         volumeMounts:
         - mountPath: /work
           name: bootstrap-manifests
+      - args:
+        - -c
+        - |
+          #!/bin/sh
+          cd /tmp
+          mkdir input output manifests
+
+          if [ ! -f /tmp/manifests/99_feature-gate.yaml ]; then
+            touch /tmp/manifests/99_feature-gate.yaml
+            cat <<EOF >/tmp/manifests/99_feature-gate.yaml
+          $(FEATURE_GATE_YAML)
+          EOF
+          fi
+
+          /usr/bin/authentication-operator render \
+            --asset-output-dir /tmp/output \
+            --rendered-manifest-dir=/tmp/manifests \
+            --cluster-profile=ibm-cloud-managed \
+            --payload-version=$(PAYLOAD_VERSION)
+
+          cp /tmp/output/* /work
```

Comment:
why don't we need to copy /tmp/manifests

Reply:
I didn't have it copy `/tmp/manifests` because, for now, the cluster-authentication-operator manifest renderer does not actually read in the feature gates file or write to it. There is no guarantee this doesn't change in the future, but I suspect it is not very likely to change.

Because of this, the feature gates file that the original manifest rendering step would produce would sometimes get overwritten, leading to some failures to create a HostedCluster.

Instead of continuing to copy the manifests over and trying to handle this edge case now, it seemed reasonable to defer that handling until the cluster-authentication-operator manifest renderer actually needs to read and write feature gates to a file.